src/blocks/notify.rs

```diff
@@ -31,6 +33,7 @@ pub struct Notify {
 #[serde(deny_unknown_fields)]
 pub struct NotifyConfig {
     /// Format string for displaying phone information.
+    /// "phone"?
```

Comment:
Copy paste error :laughing:  Can just delete the comment since we all know what the format string is for.